lib/migrate.js

```diff
@@ -69,20 +70,26 @@ function wait(connection, options) {
   - environment variables
   - /database.json
  */
-function getConfig(root) {
+function getConfig(root, env) {
+  dotenv.load({
+    silent: true
+  });
   nconf
-    .argv()
-    .env()
+    .env("_")
     .file({ file: path.join(root, 'database.json') });
-  var config = {
-    host: nconf.get('host'),
-    port: nconf.get('port'),
-    db: nconf.get('db'),
-    discovery: Boolean(nconf.get('discovery')) || false,
-    timeout: nconf.get('timeout') || (5 * 60),
-    authKey: nconf.get('authKey'),
-    ssl: nconf.get('ssl')
+  var config;
+  config = {
+    host: nconf.get("RETHINK:HOST"),
+    port: nconf.get("RETHINK:PORT") || 28015,
+    db: nconf.get("RETHINK:DB"),
+    timeout: nconf.get("RETHINK:TIMEOUT") || 60,
+    authKey: nconf.get("RETHINK:AUTHKEY")
   };
```

Comment:
discovery and ssl were never used.  I don't even know what they would mean.  The default timeout is getting changed back, not intentional.